python/cugraph/traversal/bfs.py

```diff
@@ -72,3 +73,59 @@ def bfs(G, start, return_sp_counter=False):
         df["predecessor"].fillna(-1, inplace=True)
 
     return df
+
+
+def bfs_edges(G, source, reverse=False, depth_limit=None, sort_neighbors=None,
+              return_sp_counter=False):
+    """
+    Find the distances and predecessors for a breadth first traversal of a
+    graph.
+
+    Parameters
+    ----------
+    G : cugraph.graph or NetworkX.Graph
+        graph descriptor that contains connectivity information
+    source : Integer
+        The starting vertex index
+    reverse : boolean
+        If a directed graph, then process edges in a reverse direction
+        Currently not implemented
```

Comment:
Same here, we could throw "not implemented" errors when these parameters are passed. Otherwise, the risk is that people will fill issues saying the parameter doesn't work.

Reply:
we really need to fix the code so that those arguments are handled.  I welcome people submitting issues since it will motivate fixing and let us know that it is being used.

Reply:
fix